lib/sensors.robot

```diff
@@ -0,0 +1,121 @@
+*** Settings ***
+Documentation       This library defines keywords for reading sensor data from
+...                 devices. It might need to be mostly scrapped and implemented as keywords
+...                 defined in the platform configs if it turns out that this process
+...                 differs too much depending on the platform.
+
+Resource            terminal.robot
+
+
+*** Keywords ***
+Prepare Sensors
+    [Documentation]    Do any preparation work needed for accessing sensors
+
+    # Might only do this when any method is said to be lm-sensors.
+
+    Import Variables    ${CURDIR}/../platform-configs/${SENSORS_CONFIG_FILE}
+    ${cpu_temperature_measurement_method}=    Get From Dictionary    ${CPU_TEMPERATURE_MEASUREMENT}    method
+    ${fan_pwm_measurement_method}=    Get From Dictionary    ${FAN_PWM_MEASUREMENT}    method
+    ${fan_rpm_measurement_method}=    Get From Dictionary    ${FAN_RPM_MEASUREMENT}    method
+
+    ${lm_sensors_used}=    Catenate
+    ...    '''${cpu_temperature_measurement_method}''' == '''system76-acpi''' or
+    ...    '''${cpu_temperature_measurement_method}''' == '''lm-sensors''' or
+    ...    '''${fan_rpm_measurement_method}''' != '''lm-sensors''' or
```

Comment:
This looks weird. If we don't use lm-sensors for fan RPM measurement, then we use lm-sensors anyway?